### ISSUE TITLE
Incorrect PolylineGeometry drawing results

### ISSUE THREAD

**Issue by zouyaoji** [CesiumGS/cesium#10620]
When sceneMode is SceneMode.SCENE2D and mapMode2D is MapMode2D.ROTATE, the line added by PolylineGeometry will inexplicably have an extra line pointing to the origin (this description may not be accurate).

Sandcastle example:

https://sandcastle.cesium.com/#c=3VVda9swFP0rIi9NwFVs2ZY/moaNjI3BygIte5n3oNhKIypLRlaSpiX/ffJXnbTpNih9WSDm6uqec690j69TKUoNNoxuqQKXQNAtmNGSrXP4o/YNk0Far2dSaMIEVcnAAo+JAKBMqaBXMqMxQFblyElRLdGnGNiJ2I8uEpHW9IUsmWbGNBl+VpGP9zFwwgCHIQxw5IYojAIfW2AXg3Psuk4UOhD7YeCYRWSBhxgEnm2HCNqeG/k2wjjaWx2Rh8IwcjF0PIRdL/C9hsdzkBf6PjQ4L4iwH9Q858hEIyc07sD1fMf3eqJzz2S2MYaRh1zbDVy3Yar4bQNxIuwig0ANk4tDjFAE/RDbrm8HB0xvPVsifvWXp1hubm9D55LvbqU47tK82x3WPbmlMqda7b4aJBGmc/Fh8Jdnuw2mR8Vd4EwWnAii2pQdDi6VzOddMzs06Psb9yY0ahhScDk9LGBGlDYWEe6QwnsLULirHg+jkdVw7TuDZTFIBkWTPxlUznaPlDuRrpQUcm3yLQkvaePnXG7nLL1j4jYGWq1bd1FQoqrTHl3FFdFUMcI/Pm13x1mSlH6WaktUdkADgKIio+paG2AMno6erjk/WAJABVlwmrWVdf691VkZLfTqhpY6/juovZMWm7clnzxH34wlWSiWgqOi9K6gxnM2k1yqM6vfWAu2lCovj8PNsapAcJSpxg4daFugftitNepx++eVj5q+tcPgpKK5mSnvKulD5jZfB3oPBQOwZZledTOx+hGV3pgG1FPxFZFXVb1F5SknZcmWLCVV6U029Cf9d1fx+nvwH8it+azB+kMFn1RXQpJlwxcirCD/DDBDqYofWINkcFFl7f4ATEq943Ta1faB5YVUGqwVH0I41jQ3o9VIaLxYp3dUw7QsRxctcnwInWRsYxRyeeID3LTb7CzN9LlmD0Y608nYxL+Ackkyo5TvG6o42VVhK2f6rXFCCCdjszyN1FLyBVHPmH8D

**Comment by ggetz** [comment]
Thanks for the report @zouyaoji! Is this what you're seeing?

<img width="804" alt="image" src="https://user-images.githubusercontent.com/4439461/185203923-6b85f32a-ad5d-4b70-a825-640f319f59d2.png">

**Comment by zouyaoji** [comment]
Yes, I don't know how the line in the red box came from. It only appears when sceneMode is 2 and mapMode2D is 0.

![image](https://user-images.githubusercontent.com/24430116/185275387-a22d1375-7e1c-4f0a-8b83-97f42c470bf4.png)
